interfaces/builtin/browser_support.go

```diff
@@ -123,11 +124,18 @@ const browserSupportConnectedPlugAppArmorWithSandbox = `
 /run/udev/data/+usb:[0-9]* r,
 
 # experimental
```

Comment:
@zyga: it would require combing the chrome sources to add those comments. 'experimental' comes from the upstream kernel documentation (https://github.com/torvalds/linux/blob/master/Documentation/admin-guide/devices.txt) and they seem to change from kernel to kernel. Eg, I had different access with a 4.4 kernel, a 4.8 kernel and a 4.9 kernel (but this might've been due to a new chrome). Could I find this information? Sure, but we didn't require it before and IMHO it shouldn't be required now.